lib/facter/sensu_puppet_facts.rb

```diff
@@ -0,0 +1,33 @@
+require 'puppet'
+
+module PuppetFacts
```

Comment:
does `PuppetFacts` come from puppet or from a gem?

Reply:
It's defined in that file, I should have probably used `SensuPuppetFacts` to match file name.

Reply:
Aha, let's do that to make it more apparent.